papaya-core/src/main/java/pl/edu/agh/papaya/model/Notification.java

```diff
@@ -23,10 +23,6 @@
     @JoinColumn(nullable = false)
     private Sprint sprint;
 
-    @ManyToOne
```

Comment:
Podjęłam Decyzję Projektową, że to tutaj nie jest potrzebne - wszyscy w projekcie dostają powiadomienia w tym samym momencie (przypominających nie dostają, jeśli już złożyli deklarację, ale moment jest ten sam dla całego projektu)